docs/install.md

```diff
@@ -19,8 +18,14 @@ stream.
   being installed.
 - 'manager' and 'master' are used interchangeably in the steps below.
 - To upgrade SAT, execute the pre-installation, installation, and post-installation
-  procedures for a newer distribution. The newly installed version will become
-  the default.
+procedures for a newer distribution. The newly installed version will become
+the default.
+
+    In SAT 2.4 and newer, you can instead upgrade the product stream by using the
+    Compute Node Environment (CNE) installer. It is recommended that you upgrade
+    SAT with the CNE installer because the process is both automated and logged
+    to help you save time. See [SAT Upgrade with CNE
+    Installer](cne_install.md) for more information.
```

Comment:
There clearly is something wrong with the spell check not understanding that this is a link, but if you change the formatting as follows it will pass the spell check:
```suggestion
    to help you save time. See [SAT Upgrade with CNE Installer](cne_install.md)
    for more information.
```